### PR TITLE
Add contracts_lite_vendor release

### PR DESCRIPTION
**Pull request by togaen**

### CODE REVIEW

**Review by mabelzhang — COMMENTED** [review]
Looks good.
Is it me or can this release be done with Bloom?

**Comment by togaen** [comment]
> Is it me or can this release be done with Bloom?

@mabelzhang Is that a question for me? I did use Bloom, but I couldn't get it to generate the PR; I had to do that manually.

**Review by jacobperron — APPROVED** [review]

File: dashing/distribution.yaml [hunk 1]
```diff
@@ -533,6 +533,21 @@ repositories:
       url: https://github.com/ros2/console_bridge_vendor.git
       version: dashing
     status: maintained
+  contracts_lite_vendor:
```
Comment (jacobperron) [inline dashing/distribution.yaml:536]: Why add a "_vendor" suffix? It looks like the released CMake package is simply called `contracts_lite`.

Comment (togaen) [reply dashing/distribution.yaml:536]: @jacobperron `contracts_lite` started life as a module in the Autoware.Auto project; that suffix is a convention for these kinds of wrapper packages.

**Pull request merged**